tasks/snyk/readme.md

```diff
@@ -38,3 +38,10 @@ Run the Snyk task following the guidelines on the main [toolkit help page](https
 | include_license | boolean | false | retrieve license issues? |
 | kenna_connector_id | integer | false | If set, we'll try to upload to this connector |
 | output_directory | filename | false | Will alter default filename for output. Path is relative to #{$basedir} |
+
+
+  docker run -it --rm \
+  -v ~/Desktop/toolkit_input:/opt/app/toolkit/input \
+  -v ~/Desktop/toolkit_output:/opt/app/toolkit/output \
+  -t toolkit:latest task=snyk:snyk_api_token=8535xxxjM3464:kenna_connector_id=156222
```

Comment:
Did you want to obscure the connector ID as well?

Reply:
I don't think it matters but I changed i anyhow :-)